service/deploy.py

```diff
@@ -138,7 +138,7 @@ def deploy_unmount_volume(instance_ip, username, instance_id,
 
     #lsof returns 1 on success _and_ failure, so combination of 'rc' and 'stdout' is required
     if lsof_rc != 0 and lsof_stdout != "":
-        _raise_lsof_playbook_failure(lsof_rc, lsof_stdout)
```

Comment:
I don't understand how the changes in this file relate to the rest of the PR

Reply:
This was an error uncovered by running `prospector` during the `travis.ci tests` as part of PR building. It was fixed in an effort to make the PR review-ready.